clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -533,28 +788,156 @@
       "END-EVALUATE",
       ""
     ],
-    "description": "EVALUATE"
+    "description": "Use the EVALUATE verb in place of a set of IF/ELSE constructs. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
+  "EVALUATE FALSE": {
+    "prefix": "EVALUATE FALSE",
+    "body": [
+      "EVALUATE FALSE",
+      "\tWHEN ${1:VAR-A} = ${2:VAR-B}",
+      "\t\tMOVE \"${1:VAR-A} - ${2:VAR-B} NOT EQUAL\" TO DISP-OUT.",
+      "\tWHEN ${3:VAR-C} = ${4:VAR-D}",
+      "\t\tMOVE \"${3:VAR-C} - ${4:VAR-D} NOT EQUAL\" TO DISP-OUT.",
+      "END-EVALUATE.",
+      "$0"
+    ],
+    "description": "Use EVALUATE instead of a series of nested IF statements to test multiple conditions and specify distinct actions for each. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
   },
   "FILE": {
     "prefix": "FILE",
     "body": ["FILE ${1:item}"],
     "description": "FILE item"
   },
-  "FIRST": {
+  "FILE SECTION": {
+    "prefix": "FILE SECTION",
+    "body": [
+      "DATA DIVISION.",
```

Comment:
The division statement should be removed (also in the snippets below).
I think it is reasonable to assume that the user is already working with a valid cobol program and just wants to add new statements.